internal/cliwrapper/cliwrapper.go

```diff
@@ -116,13 +115,13 @@ func (p *cliWrapper) PullModule(fullModuleName string) error {
 	// authentication causing pip to hang because pip calls `git clone` in
 	// a subprocess.
 	cmdPip.Env = append(os.Environ(), "GIT_TERMINAL_PROMPT=0")
-	
+
 	output, err := cmdPip.Output()
 	if len(output) > 0 {
 		p.logger.Debugf("pip install stdout: %s", output)
 	}
 	if err != nil {
-		return fmt.Errorf("error pip installing (%w)", err)
+		return fmt.Errorf("error pip installing %s (%w)", fullModuleName, err)
 	}
```

Comment:
"error pip installing" reads funny.  Consider using something like "error when pip-installing" or "Pip installation error", instead.

Reply:
i like the way it reads funny